src/scry.cr

```diff
@@ -39,6 +39,15 @@ module Scry
   ensure
     Log.logger.close unless Log.logger.nil?
   end
+
+  def self.process_error(request : Protocol::RequestMessage, error)
```

Comment:
I'd suggest `make_error`, you're not really processing the error here

Reply:
The reason I made it `process_error` is because the Error has already happened and now we are just handling it, and depending on the type of message we do something different.  In the case of a error for a `NotificationMessage` we are not making an error message.  Maybe `handle_error` would be better?

Reply:
Yes `handle_error` seems good, thank you